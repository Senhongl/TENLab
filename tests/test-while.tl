a = 1;
b = 2;

while (a) {
    if (b == 10) {
        a = 0;
    }
    print(b);
    b = b + 1;
}