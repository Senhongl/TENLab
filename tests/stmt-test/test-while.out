2
[ CPUIntType{} ]
3
[ CPUIntType{} ]
4
[ CPUIntType{} ]
5
[ CPUIntType{} ]
6
[ CPUIntType{} ]
7
[ CPUIntType{} ]
8
[ CPUIntType{} ]
9
[ CPUIntType{} ]
10
[ CPUIntType{} ]
